distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,298 @@
+
+==== Keystore and Certificate Management
+
+${branding} uses certificates in three ways:
+
+. Ensuring the privacy and integrity of message sent or received over a network.
+. Transmitting and receiving encrypted messages.
```

Comment:
This bullet point is redundant with the first bullet point, so delete 
     "Transmitting and receiving encrypted messages."

Then change "three" to "two" on line 4.